inc/vcf/file_structure.hpp

```diff
@@ -496,6 +517,13 @@ namespace ebi
         void check_sample_alleles_range(std::string const & allele, long ploidy) const;
 
         /**
+         * Checks that fields have no duplicates
```

Comment:
The function is so generic that it doesn't refer to a field at all, so maybe documentation like this?

`Checks that a list contains no duplicates`